Source/Wit/Private/TTS/Cache/Memory/TtsMemoryCache.cpp

```diff
@@ -95,6 +95,16 @@ void UTtsMemoryCache::RemoveAllClips()
 	ClipSettingsArray.Empty();
 }
 
+/**
+ * Gets number of clips in cache
+ *
+ * @return number of clips in cache
+ */	
+int32 UTtsMemoryCache::GetCacheClipNum() const
```

Comment:
Looks fine. But perhaps we can name it similar to GetUsedCacheSizeInKilobytes() for consistency? So something like GetUsedCacheClipCapacity()

Reply:
Great names! I spend 20 minutes to find a better name, but didn't work it out.  I like your names.  
I will use GetUsedCacheClipCapacity()
Thanks